[Time to Visualize] Enable by Default

ThomThomson opened the pull request:
## Summary
A major piece of https://github.com/elastic/kibana/issues/51318
Closes https://github.com/elastic/kibana/issues/52682

Release Notes: Enable "by value embeddables" (dashboard.allowByValueEmbeddables) by default

All progress on the Time to Visualize project has been hidden behind the flag `dashboard.allowByValueEmbeddables`. This PR changes the default value of that config option to true.

Documentation will be added in a followup.

### Checklist

Delete any items that are not applicable to this PR.

- [ ] [Documentation](https://www.elastic.co/guide/en/kibana/master/development-documentation.html) was added for features that require explanation or tutorials
- [x] [Unit or functional tests](https://www.elastic.co/guide/en/kibana/master/development-tests.html) were updated or added to match the most common scenarios

### For maintainers

- [ ] This was checked for breaking API changes and was [labeled appropriately](https://www.elastic.co/guide/en/kibana/master/contributing.html#kibana-release-notes-process)

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/98971/)
* Commit: f041badae4aba80add1429fef99030c05649bb9e
* This comment will update when the build is complete

### Failed CI Steps
* [Execute kibana-ciGroup9](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/98971/execution/node/537/log)
* [Execute kibana-ciGroup12](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/98971/execution/node/555/log)

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/98971/","number":"98971","commit":"f041badae4aba80add1429fef99030c05649bb9e"}]}
PIPELINE-->

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99024/)
* Commit: 3f14c6ffbef63a4f2d521dc33bdf2f9b3121ab98
* This comment will update when the build is complete

### Failed CI Steps
* [Execute oss-firefox](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99024/execution/node/483/log)
* [Execute kibana-ciGroup2](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99024/execution/node/495/log)

### History
* :broken_heart: [Build #98971](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/98971/) failed f041badae4aba80add1429fef99030c05649bb9e

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/98971/","number":"98971","commit":"f041badae4aba80add1429fef99030c05649bb9e"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99024/","number":"99024","commit":"3f14c6ffbef63a4f2d521dc33bdf2f9b3121ab98"}]}
PIPELINE-->

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99033/)
* Commit: 676f9717496772cb020e3c19ccb6af4ed6135e03
* This comment will update when the build is complete

### Failed CI Steps
* [Execute kibana-ciGroup9](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99033/execution/node/537/log)
* [Execute kibana-ciGroup11](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99033/execution/node/549/log)

### History
* :broken_heart: [Build #99024](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99024/) failed 3f14c6ffbef63a4f2d521dc33bdf2f9b3121ab98
* :broken_heart: [Build #98971](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/98971/) failed f041badae4aba80add1429fef99030c05649bb9e

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/98971/","number":"98971","commit":"f041badae4aba80add1429fef99030c05649bb9e"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99024/","number":"99024","commit":"3f14c6ffbef63a4f2d521dc33bdf2f9b3121ab98"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99033/","number":"99033","commit":"676f9717496772cb020e3c19ccb6af4ed6135e03"}]}
PIPELINE-->

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99041/)
* Commit: 9b596a1e3cc07e1983fca59720184bb09cb7e51b
* This comment will update when the build is complete

### Failed CI Steps
* [Execute xpack-kibana-ciGroup2](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99041/execution/node/613/log)
* [Execute xpack-kibana-ciGroup2](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99041/execution/node/803/log)

### History
* :broken_heart: [Build #99033](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99033/) failed 676f9717496772cb020e3c19ccb6af4ed6135e03
* :broken_heart: [Build #99024](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99024/) failed 3f14c6ffbef63a4f2d521dc33bdf2f9b3121ab98
* :broken_heart: [Build #98971](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/98971/) failed f041badae4aba80add1429fef99030c05649bb9e

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/98971/","number":"98971","commit":"f041badae4aba80add1429fef99030c05649bb9e"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99024/","number":"99024","commit":"3f14c6ffbef63a4f2d521dc33bdf2f9b3121ab98"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99033/","number":"99033","commit":"676f9717496772cb020e3c19ccb6af4ed6135e03"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99041/","number":"99041","commit":"9b596a1e3cc07e1983fca59720184bb09cb7e51b"}]}
PIPELINE-->

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99136/)
* Commit: b33de6e174e579dcbf3f89e02bd09fada41d7fb5

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/a77f0a63c324cdc2de7afa4611783e511042be7b) | [after](https://github.com/elastic/kibana/commit/b33de6e174e579dcbf3f89e02bd09fada41d7fb5) | diff |
| --- | --- | --- | --- |
| `visualize` | 132.3KB | 132.3KB | +25.0B |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/a77f0a63c324cdc2de7afa4611783e511042be7b) | [after](https://github.com/elastic/kibana/commit/b33de6e174e579dcbf3f89e02bd09fada41d7fb5) | diff |
| --- | --- | --- | --- |
| `presentationUtil` | 25.3KB | 25.4KB | +92.0B |

### History
* :broken_heart: [Build #99041](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99041/) failed 9b596a1e3cc07e1983fca59720184bb09cb7e51b
* :broken_heart: [Build #99033](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99033/) failed 676f9717496772cb020e3c19ccb6af4ed6135e03
* :broken_heart: [Build #99024](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99024/) failed 3f14c6ffbef63a4f2d521dc33bdf2f9b3121ab98
* :broken_heart: [Build #98971](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/98971/) failed f041badae4aba80add1429fef99030c05649bb9e

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/98971/","number":"98971","commit":"f041badae4aba80add1429fef99030c05649bb9e"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99024/","number":"99024","commit":"3f14c6ffbef63a4f2d521dc33bdf2f9b3121ab98"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99033/","number":"99033","commit":"676f9717496772cb020e3c19ccb6af4ed6135e03"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99041/","number":"99041","commit":"9b596a1e3cc07e1983fca59720184bb09cb7e51b"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99136/","number":"99136","commit":"b33de6e174e579dcbf3f89e02bd09fada41d7fb5"}]}
PIPELINE-->

majagrubic commented on x-pack/test/functional/page_objects/lens_page.ts:
```diff
@@ -287,6 +293,22 @@ export function LensPageProvider({ getService, getPageObjects }: FtrProviderCont
+        await label.click();
+
+        if (dashboardId) {
+          // TODO - selecting an existing dashboard
```
I'm a bit confused with the flow here: is the goal to ensure just that the all dashboard selectors exist, without actually adding the Lens visualization to the dashboard?

ThomThomson replied:
as part of https://github.com/elastic/kibana/pull/83140, a new selector was added to the visualize save modal to allow users to add by value directly to a dashboard from visualize. That also means that in order to save it to the library, the user needs to select the `add to library` option on the dashboard selector.

What I've added here is a default path that says 'if the dashboard selection section exists, and we haven't been asked to add it directly to a dashboard, select the `add to library` option. This prevents a number of test failures - and you can see a similar implementation in the Visualize page.

lukeelmers replied:
> What I've added here is a default path that says 'if the dashboard selection section exists, and we haven't been asked to add it directly to a dashboard, select the add to library option. This prevents a number of test failures - and you can see a similar implementation in the Visualize page.

nit: It might be helpful to extract this logic into another helper function that could be pulled into the lens & SO tagging functional tests. It feels like the type of implementation detail that other apps shouldn't necessarily need to concern themselves with when writing tests.

e.g. something like this could potentially be used everywhere:
```js
await saveToNewDashboard();
await saveToExistingDashboard('my-dashboard-id');
await saveToLibrary();
```

elasticmachine commented:
Pinging @elastic/kibana-presentation (Team:Presentation)

majagrubic commented on src/plugins/dashboard/config.ts:
```diff
@@ -20,7 +20,7 @@
 
 export const configSchema = schema.object({
-  allowByValueEmbeddables: schema.boolean({ defaultValue: false }),
+  allowByValueEmbeddables: schema.boolean({ defaultValue: true }),
```
🎉

majagrubic replied:
Just curious, is the plan to enable this in 7.12 and then delete everything in 7.13? Or are you also planning for a cleanup in a separate PR?

ThomThomson replied:
That's still up in the air - because removing the ability to turn the feature off _could_ constitute a breaking change. Right now the consensus seems to be removing references to it in 8.0.0

clintandrewhall replied:
We'll craft a plan to *at least* deprecate with a warning in 8.0 for removal in a follow-on minor, if not remove outright.  It kind of depends on feedback from the field.

clintandrewhall reviewed (approved):
Fantastic... LGTM! 🚢 

Congrats @elastic/kibana-presentation

lukeelmers reviewed (approved):
Updates to core tests LGTM, just had one suggestion of a possible way to clean things up.

ThomThomson commented:
> nit: It might be helpful to extract this logic into another helper function

Good call! I've cleaned this up by extracting all of the setting of save modal values into a new function in visualize page. Updated everywhere else it's used to use the new function.

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99282/)
* Commit: 7a9176951629be316f8083ceae654ab4626c58a2
* This comment will update when the build is complete

### Failed CI Steps
* [Execute kibana-ciGroup9](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99282/execution/node/537/log)
* [Execute kibana-ciGroup12](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99282/execution/node/555/log)

### History
* :green_heart: [Build #99136](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99136/) succeeded b33de6e174e579dcbf3f89e02bd09fada41d7fb5
* :broken_heart: [Build #99041](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99041/) failed 9b596a1e3cc07e1983fca59720184bb09cb7e51b
* :broken_heart: [Build #99033](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99033/) failed 676f9717496772cb020e3c19ccb6af4ed6135e03
* :broken_heart: [Build #99024](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99024/) failed 3f14c6ffbef63a4f2d521dc33bdf2f9b3121ab98
* :broken_heart: [Build #98971](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/98971/) failed f041badae4aba80add1429fef99030c05649bb9e

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/98971/","number":"98971","commit":"f041badae4aba80add1429fef99030c05649bb9e"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99024/","number":"99024","commit":"3f14c6ffbef63a4f2d521dc33bdf2f9b3121ab98"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99033/","number":"99033","commit":"676f9717496772cb020e3c19ccb6af4ed6135e03"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99041/","number":"99041","commit":"9b596a1e3cc07e1983fca59720184bb09cb7e51b"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99136/","number":"99136","commit":"b33de6e174e579dcbf3f89e02bd09fada41d7fb5"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99282/","number":"99282","commit":"7a9176951629be316f8083ceae654ab4626c58a2"}]}
PIPELINE-->

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99289/)
* Commit: 068aab16787ac25f247d97f164a7558e620549d0

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/a77f0a63c324cdc2de7afa4611783e511042be7b) | [after](https://github.com/elastic/kibana/commit/068aab16787ac25f247d97f164a7558e620549d0) | diff |
| --- | --- | --- | --- |
| `visualize` | 132.3KB | 132.3KB | +25.0B |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/a77f0a63c324cdc2de7afa4611783e511042be7b) | [after](https://github.com/elastic/kibana/commit/068aab16787ac25f247d97f164a7558e620549d0) | diff |
| --- | --- | --- | --- |
| `presentationUtil` | 25.3KB | 25.4KB | +92.0B |

### History
* :broken_heart: [Build #99282](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99282/) failed 7a9176951629be316f8083ceae654ab4626c58a2
* :green_heart: [Build #99136](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99136/) succeeded b33de6e174e579dcbf3f89e02bd09fada41d7fb5
* :broken_heart: [Build #99041](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99041/) failed 9b596a1e3cc07e1983fca59720184bb09cb7e51b
* :broken_heart: [Build #99033](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99033/) failed 676f9717496772cb020e3c19ccb6af4ed6135e03
* :broken_heart: [Build #99024](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99024/) failed 3f14c6ffbef63a4f2d521dc33bdf2f9b3121ab98

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99024/","number":"99024","commit":"3f14c6ffbef63a4f2d521dc33bdf2f9b3121ab98"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99033/","number":"99033","commit":"676f9717496772cb020e3c19ccb6af4ed6135e03"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99041/","number":"99041","commit":"9b596a1e3cc07e1983fca59720184bb09cb7e51b"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99136/","number":"99136","commit":"b33de6e174e579dcbf3f89e02bd09fada41d7fb5"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99282/","number":"99282","commit":"7a9176951629be316f8083ceae654ab4626c58a2"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99289/","number":"99289","commit":"068aab16787ac25f247d97f164a7558e620549d0"}]}
PIPELINE-->

ThomThomson commented:
@elasticmachine merge upstream

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99424/)
* Commit: bc94480d2bb29d286f34b73d4ad70f6222091735

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/53f4b21a8158d0becbd7874f75c0d3a85ceefbe5) | [after](https://github.com/elastic/kibana/commit/bc94480d2bb29d286f34b73d4ad70f6222091735) | diff |
| --- | --- | --- | --- |
| `visualize` | 132.3KB | 132.3KB | +25.0B |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/53f4b21a8158d0becbd7874f75c0d3a85ceefbe5) | [after](https://github.com/elastic/kibana/commit/bc94480d2bb29d286f34b73d4ad70f6222091735) | diff |
| --- | --- | --- | --- |
| `presentationUtil` | 25.3KB | 25.4KB | +92.0B |

### History
* :green_heart: [Build #99289](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99289/) succeeded 068aab16787ac25f247d97f164a7558e620549d0
* :broken_heart: [Build #99282](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99282/) failed 7a9176951629be316f8083ceae654ab4626c58a2
* :green_heart: [Build #99136](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99136/) succeeded b33de6e174e579dcbf3f89e02bd09fada41d7fb5
* :broken_heart: [Build #99041](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99041/) failed 9b596a1e3cc07e1983fca59720184bb09cb7e51b
* :broken_heart: [Build #99033](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99033/) failed 676f9717496772cb020e3c19ccb6af4ed6135e03

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99033/","number":"99033","commit":"676f9717496772cb020e3c19ccb6af4ed6135e03"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99041/","number":"99041","commit":"9b596a1e3cc07e1983fca59720184bb09cb7e51b"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99136/","number":"99136","commit":"b33de6e174e579dcbf3f89e02bd09fada41d7fb5"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99282/","number":"99282","commit":"7a9176951629be316f8083ceae654ab4626c58a2"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99289/","number":"99289","commit":"068aab16787ac25f247d97f164a7558e620549d0"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99424/","number":"99424","commit":"bc94480d2bb29d286f34b73d4ad70f6222091735"}]}
PIPELINE-->

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/113496/)
* Commit: bc94480d2bb29d286f34b73d4ad70f6222091735
* This comment will update when the build is complete

### Failed CI Steps
* [Setup Build Environment and Dependencies](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/113496/execution/node/106/log)
* [Setup Build Environment and Dependencies](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/113496/execution/node/112/log)

### History
* :green_heart: [Build #99424](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99424/) succeeded bc94480d2bb29d286f34b73d4ad70f6222091735
* :green_heart: [Build #99289](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99289/) succeeded 068aab16787ac25f247d97f164a7558e620549d0
* :broken_heart: [Build #99282](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99282/) failed 7a9176951629be316f8083ceae654ab4626c58a2
* :green_heart: [Build #99136](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99136/) succeeded b33de6e174e579dcbf3f89e02bd09fada41d7fb5
* :broken_heart: [Build #99041](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99041/) failed 9b596a1e3cc07e1983fca59720184bb09cb7e51b

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99041/","number":"99041","commit":"9b596a1e3cc07e1983fca59720184bb09cb7e51b"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99136/","number":"99136","commit":"b33de6e174e579dcbf3f89e02bd09fada41d7fb5"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99282/","number":"99282","commit":"7a9176951629be316f8083ceae654ab4626c58a2"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99289/","number":"99289","commit":"068aab16787ac25f247d97f164a7558e620549d0"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/99424/","number":"99424","commit":"bc94480d2bb29d286f34b73d4ad70f6222091735"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/113496/","number":"113496","commit":"bc94480d2bb29d286f34b73d4ad70f6222091735"}]}
PIPELINE-->
